home-assistant-plugin/custom_components/xknx/__init__.py

```diff
@@ -327,6 +327,7 @@ async def telegram_received_cb(self, telegram):
             {
                 "address": str(telegram.destination_address),
                 "data": telegram.payload.value,
+                "source": str(telegram.source_address),
```

Comment:
Should we use "destination_address" instead of "address" here too?

Reply:
We should use a consistent naming scheme here:

source_address and destination_address sounds reasonable.